src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 0;
+
+  const getSum = function(num1) {
+    count++;
+
+    return function(num2) {
```

Comment:
you can just use arrow functions
```suggestion
  const getSum = (num1) => {
    count++;

    return (num2) =>  {
```